homepage/homepage/app/(docs)/docs/[framework]/[...slug]/groups/inheritance.mdx

```diff
@@ -0,0 +1,161 @@
+import { ContentByFramework, CodeGroup } from '@/components/forMdx'
+
+export const metadata = { title: "Group inheritance" };
+
+# Group Inheritance
+
+Groups can inherit members from other groups using the `extend` method. 
+
+When a group extends another group, members of the parent group will become automatically part of the child group.
+
+## Basic Usage
+
+Here's how to extend a group:
+
+<CodeGroup>
+```typescript
+const playlist = Group.create();
```

Comment:
```suggestion
const playlistGroup = Group.create();
```

Can we call all of them ..Group please? And maybe show how the groups are used as an owner on something?

I know it's a bit tedious but as-is it's confusing